flow/raster_cache.h

```diff
@@ -298,50 +183,51 @@ class RasterCache {
    */
   int access_threshold() const { return access_threshold_; }
 
+  bool GenerateNewCacheInThisFrame() const {
+    // Disabling caching when access_threshold is zero is historic behavior.
+    return access_threshold_ != 0 &&
+           picture_cached_this_frame_ + display_list_cached_this_frame_ <
+               picture_and_display_list_cache_limit_per_frame_;
+  }
+
+  /**
+   * @brief if the item has been cached return true.
+   * We mean the item will show in the screen, and will create a cache entry if
+   * the item doesn't create cache.
+   * the return value is the entry access count, so if a new entry it will be 1
+   */
+  int MarkSeen(const RasterCacheKeyID& id, const SkMatrix& matrix) const;
```

Comment:
The return value no longer indicates if the cache entry is present like the comment says. That's probably OK as I don't think the callers look to it for that information. I would probably just have the comment say that it returns the number of times the entry has been hit since it was created.

Reply:
Done